openasip/src/bintools/Compiler/riscv-tdgen/riscv-tdgen.cc

```diff
@@ -38,12 +38,15 @@
 int main(int argc, char* argv[]) {
     std::string outputDir;
     std::string adfPath;
+    std::string rocc_str;
+    bool rocc;
 
     // Check if the correct number of arguments is provided
-    if (argc != 5) {
+    if (argc != 7) {
         std::cout << "Usage: riscv-tdgen" << std::endl
                   << "   -o Output directory." << std::endl
-                  << "   -a ADF path." << std::endl;
+                  << "   -a ADF path." << std::endl
+                  << "   -r 'T':Enable ROCC encodings, OR 'F'" << std::endl;
```

Comment:
Can you add a default value to this? We want to run this command from the command line, and setting the coprocessor encoding type each time should not be necessary.